src/TodoList.js

```diff
@@ -16,15 +17,14 @@ const todoList = [
     
   ];
   
-function TodoList () {
+function TodoList ({ todoList }) {
     return (
-        <ul>
+      <ul>
         {todoList.map((item) => (
-          <li key={item.id}>{item.title}</li>
+        <TodoListItem key={item.id} item={item} />
```

Comment:
This is totally fine as it is, but is also an area where you could use destructuring if you wanted
```suggestion
      {todoList.map(({ id, title }) => (
        <TodoListItem key={id} todoTitle={title} />
      ))}

```